documentation.md

```diff
@@ -0,0 +1,131 @@
+# strong-error-handler documentation
+
+This middleware is intended to be used in both development (debug) and production environments.
+
+## Production mode
+
+- Error responses output do not contain sensitive details or  information.
+- Only `safeFields` are generated in the output.
+
+## Debug Mode
+
+- Full error stack traces and internal details of any object passed to this module will be sent back to the client in the HTTP responses when an error occurs.
+
+## Response Format
+
+- `strong-error-handler` currently supports HTML, JSON, and plain text responses.
```

Comment:
plain text is not supported yet, see https://github.com/strongloop/strong-error-handler/issues/6